packages/amplify-provider-awscloudformation/src/push-resources.ts

```diff
@@ -87,6 +87,7 @@ const ApiServiceNameElasticContainer = 'ElasticContainer';
 
 const optionalBuildDirectoryName = 'build';
 const cfnTemplateGlobPattern = '*template*.+(yaml|yml|json)';
+const nestedStackTemplateGlobPattern = 'stacks/*.+(yaml|yml|json)';
```

Comment:
See [this](https://github.com/aws-amplify/amplify-category-api/blob/08b4f168ecb88b5d829f884fb10df3610634643f/packages/amplify-category-api/src/graphql-transformer/utils.ts#L297)